zerver/lib/markdown/__init__.py

```diff
@@ -276,8 +276,8 @@ def image_preview_enabled(message: Optional[Message]=None,
     return realm.inline_image_preview
 
 def list_of_tlds() -> List[str]:
-    # HACK we manually blacklist a few domains
-    blacklist = ['PY\n', "MD\n"]
+    # We manually blacklist a few domains that are common as file extensions.
+    blacklist = ['PY\n', "MD\n", "MOV\n", "ZIP\n"]
```

Comment:
`.JAVA` might go on here as well.

Reply:
Yeah, good idea re: checklist.  We should also reformat the code a bit here to not require the awkward `\n`s in our blacklist.